core/packages/PackageDB.cc

```diff
@@ -102,15 +112,19 @@ const PackageInfo &PackageDB::getPackageForFile(const core::GlobalState &gs, cor
 }
 
 const PackageInfo &PackageDB::getPackageInfo(core::NameRef mangledName) const {
-    auto it = packages.find(mangledName);
-    if (it == packages.end()) {
+    auto it = packages_.find(mangledName);
+    if (it == packages_.end()) {
         return NONE_PKG;
     }
     return *it->second;
 }
 
 bool PackageDB::empty() const {
-    return packages.empty();
+    return packages_.empty();
+}
+
+const vector<core::NameRef> &PackageDB::packages() const {
```

Comment:
Question about the name of this function: Are these package namespaces?

Reply:
These are the mangled names, which I'm effectively using as refs to pass around for packages.